src/util.cpp

```diff
@@ -8,18 +8,21 @@
  * @copyright Copyright (c) 2023
  * 
  */
+
 #include "util.hpp"
 
 #include "cli/conio.hpp"
 #include "consts.hpp"
 
 #include "Particle.h"
 
+/*! \def BYTES_PER_LINE
```

Comment:
Please remove this line

Reply:
Oh sorry. It should still be
/*!
    \brief A macro that stores the the size of each line of output as a constant 16 bytes
*/
because /*! is signifying documentation, correct?

Reply:
Yes. Remember that you can try this and test it by making the change locally and building the documentation. You have access to the logs as well as the documentation output to verify for yourself whether this works or not, and if it does what you want it to do.

Reply:
Got it, I think the HTML doc looks good